applications/matter_weather_station/CMakeLists.txt

```diff
@@ -10,8 +10,9 @@ if (NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE ZDebug)
 endif()
 
-set(CONF_FILE           ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/prj_${CMAKE_BUILD_TYPE}.conf)
-set(PM_STATIC_YML_FILE  ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static.yml)
+set(CONF_FILE			  ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/prj_${CMAKE_BUILD_TYPE}.conf)
+set(PM_STATIC_YML_FILE		  ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static.yml)
+set(multiprotocol_rpmsg_CONF_FILE ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/multiprotocol_rpmsg.conf)
```

Comment:
Why do we need to override default multiprotocol_rpmsg config file here? If we just want to add three BT configs, I believe it would be better to create multiprotocol_rpmsg.conf in the child_image directory to attach it to the build as an overlay. I don't see the point of defining all configs on our own here, as it will add us additional maintenance work to track multiprotocol_rpmsg changes and align them locally.

Reply:
I prefer to define this explicitly for Thingy:53 board (the DFU functionality relates only for that board). This approach also gives application more control on used configuration (we prefer this for our applications).
If you wish, I may add this as a generic overlay for multiprotocol_rpmsg that would be used for all of the boards.

Reply:
We would prefer to keep this change as an overlay for multiprotocol_rpmsg instead of overriding defaults, as for us it is important to keep multiprotocol image configs up to date with the most recent versions. This application supports only Thingy:53 and we have no plans to expand it on the others, so there will not be a problem with lack of support for DFU capability even if you just add it into the child_image directory.